src/history.js

```diff
@@ -187,14 +187,14 @@ class History extends Emitter {
     if (this.size <= 0) {
       this.begin()
       return
-    } else if (!next) {
+    } else if (action === this.head) {
       next = this.head = parent
     } else if (action === this.root) {
       this.root = next
     }
 
     if (!action.disabled) {
-      this.reconcile(next)
+      this.reconcile(this.head)
```

Comment:
okay, so thinking I'll update this code here for removing active/non-active actions, and then also update the `checkout` logic to reconcile from the most recent shared action - that sound right?

here, in the case of an active action, would reconciliation need to start at the parent of the removed action?